server/main.jai

```diff
@@ -13,14 +13,16 @@ User_Options :: struct {
 }
 
 Server :: struct {
-    args: Args;
     quit := false;
 
-    // This is from the config file
```

Comment:
Removed because misleading: `roots` et al are discovered if not set to anything in config.